DataRepo/views/models/bst/client_interface.py

```
@@ -520,14 +562,10 @@ def reset_column_cookies(self, columns: List[Union[str, BSTBaseColumn]], name: s
             columns (List[Union[str, BSTBaseColumn]]): A list of BST columns or column names.
             name (str): The name of the cookie variable specific to the column.
         Exceptions:
-            ValueError when columns is invalid.
+            None
         Returns:
             None
         """
-        if columns is None or len(columns) == 0:
-            raise ValueError(
-                f"Invalid columns: [{columns}].  Must be a non-empty list of strs or BSTBaseColumns."
-            )
```

C. This came up in the new tests.  The error was unnecessary.  If there are no filter cookies to clear, it should just silently do nothing.